src/main/resources/application.yml

```
@@ -14,4 +14,26 @@ management:
       jmx:
         exposure.include: "*"
       web:
-        exposure.include: info, health
\ No newline at end of file
+        exposure.include: info, health
+app:
+    pdnd:
+      base-url: "\${PDND_BASE_URL:https://auth.uat.interop.pagopa.it}"
+      config:
+        audience: "\${PDND_ACCESS_TOKEN_AUDIENCE:auth.uat.interop.pagopa.it/client-assertion}"
+      anpr:
+        services:
+          c003:
+            client-id: "\${PDND_SERVICE_ANPR_C003_CLIENTID:\${PDND_SERVICE_ANPR_CLIENTID:\${PDND_SERVICE_CLIENTID:clientid}}}"
+            kid: "\${PDND_SERVICE_ANPR_C003_KID:\${PDND_SERVICE_ANPR_KID:\${PDND_SERVICE_KID:kid}}}"
+            purpose-id: "\${ANPR_PDND_CONFIGURATION_C003_PURPOSE_ID:c003purposeid}"
```

let's same naming convention of other ENV vars
```suggestion
            purpose-id: "\${PDND_SERVICE_ANPR_C003_PURPOSEID:c003purposeid}"
```